### ISSUE TITLE
$.escape should be renamed to $.raw

### ISSUE THREAD

**Issue by JLarky** [oven-sh/bun#8318]
### What is the problem this feature would solve?

$.escape does the opposite of what the name implies

### What is the feature you are proposing to solve the problem?

Deprecate `$.escape` and add `$.raw`

### What alternatives have you considered?

$.unescape? :-)

**Comment by trvswgnr** [comment]
fully agree this is confusing. i would expect escape to... well, *escape* the input.

**Comment by Jarred-Sumner** [comment]
reasonable

**Comment by zackradisic** [comment]
This is a mistake in the docs, `$.escape` actually does what is expected:

```typescript
console.log($.escape('$(foo) `bar` "baz"'))
// Prints: \$(foo) \`bar\` \"baz\"
```

If you want to pass a string as raw input, use `{ raw: 'str' }`:
```typescript
await $`echo ${{ raw: '$(foo) `bar` "baz"' }}`
```

**Comment by JLarky** [comment]
Thanks for clarification, this makes much more sense :)

**Comment by paperclover** [comment]
let's fix the documentation of this. i was actually unaware of this too

**Comment by zackradisic** [comment]
Fixed f77b217
